runtime/src/static/cpp/Config.cpp

```diff
@@ -72,19 +72,22 @@ void initializeConfig(int numThreads) {
   }
 }
 
-void initializeGlobal(int numThreads, size_t heapSize) {
+void initializeGlobal(int lowestCppTid, int numCppThreads, size_t heapSize, bool enablePCM) {
```

Comment:
there seem to be 'enablePCM' flags in multiple places in both the framework and runtime... are all these really necessary if we're using C macros to guard the PCM api calls?